### PR TITLE
INT-303 INT-301 Allow to purchase multiple phone numbers at once

### PR DESCRIPTION
**Pull request by BullDog57Rus**
Add the PhoneNumber.Name field

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -2110,88 +2063,94 @@ components:
...
+          type: integer
+        countryCode:
+          $ref: '#/components/schemas/CountryCode'
+        whatsappEnabled:
+          type: boolean
+        name:
+          type: string
+          description: The given name of the phone number
+        assignmentType:
+          $ref: '#/components/schemas/PhoneNumberAssigmentType'
```
Comment (evgeniyp) [inline talk/talk-api.yaml:2091]:
```suggestion
          $ref: '#/components/schemas/PhoneNumberAssignmentType'
```

**Review by evgeniyp — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -2110,88 +2063,94 @@ components:
...
-          enum:
-            - USER
-            - UNASSIGNED
-            - AUTO_RECEPTIONIST
-        whatsappEnabled:
-          type: boolean
-      required:
-        - id
-      description: User phone number model
+    PhoneNumberAssigmentType:
```
Comment (evgeniyp) [inline talk/talk-api.yaml:2106]:
```suggestion
    PhoneNumberAssignmentType:
```

**Review by evgeniyp — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -2110,88 +2063,94 @@ components:
...
+        - countryCode
+        - whatsappEnabled
       properties:
         id:
           $ref: '#/components/schemas/Id'
+        phoneNumber:
+          type: string
+        type:
+          $ref: '#/components/schemas/PhoneNumberType'
+        timeout:
```
Comment (aehrenthal) [inline talk/talk-api.yaml:2081]: I think a description of this field would be helpful

**Review by aehrenthal — APPROVED** [review]

File: talk/talk-api.yaml (hunk 3 above)
Comment (BullDog57Rus) [reply talk/talk-api.yaml:2057]: done :)

**Review by evgeniyp — APPROVED** [review]

**Pull request merged**